src/routes/+error.svelte

```diff
@@ -1,16 +1,26 @@
 <script lang="ts">
 	import { page } from '$app/stores';
+	import ButtonSolid from '$lib/button-solid.svelte';
+	import '../app.css';
 </script>
 
 <div
 	class="bg-brown-200 text-center min-h-[70vh] py-8 2xl:py-16 flex-col w-full h-full flex justify-center items-center"
 >
 	{#if $page.status === 400 || $page.status === 401}
-		<img class="w-4/6" src="/unauthorised.svg" alt="" />
+		<img class="w-ful" src="/unauthorised.svg" alt="" />
 	{:else}
-		<img class="w-4/6" src="/404-not-found.svg" alt="" />
+		<img class="w-full" src="/404-not-found.svg" alt="" />
 	{/if}
-	<div class="w-full h-full">
-		<span class="text-brown-900 text-xl">Status Code: {$page.status} {JSON.stringify($page.error)}</span>
+	<div class="w-full h-full flex flex-col justify-start gap-4 items-center">
+		<span class="text-brown-900 text-xl">
+			Status Code: {$page.status}
+			{JSON.stringify($page.error)}
+		</span>
+		<div class="flex gap-4 justify-center items-center">
+			<ButtonSolid>Login</ButtonSolid>
```

Comment:
we definitely need to work further on these buttons. let's add some comments for now?